lib/router.js

```diff
@@ -1,41 +1,44 @@
 FlowRouter.route('/', {
-    subscriptions: function(params) {
-        console.log("all the subscriptions goes here!");
-    },
-    action: function(params) {
-    	// Session regsitering
-    	Session.set('loggedIn', false);
-    	Session.set('screenName', '');
+  subscriptions: function(params) {
+  },
+  action: function(params) {
+    // Session regsitering
+    Session.set('loggedIn', false);
+    Session.set('screenName', '');
 
-        FlowLayout.render('layout', { template: "login" });
-    }
-});
-
-FlowRouter.route('/about', {
-    subscriptions: function(params) {
-        console.log("all the subscriptions goes here!");
-    },
-    action: function(params) {
-        FlowLayout.render('layout', { template: "page", page: "about" });
-    }
+    FlowLayout.render('layout', { template: "login" });
+  }
 });
 
 FlowRouter.route('/room', {
-    subscriptions: function(params) {
-        //Subscriptions
-        Meteor.subscribe('People');
-        Meteor.subscribe('Discussion');
-    },
-    action: function(params) {
-        FlowLayout.render('layout', { template: "room" });
-    }
-});
+  subscriptions: function(params) {
+    //Subscriptions
+    this.register('People', Meteor.subscribe('People'));
+    this.register('Discussion', Meteor.subscribe('Discussion'));
+  },
+  action: function(params) {
+    if( Meteor.user() ) {
+      screenName = Meteor.user().services.github.username;
```

Comment:
You can't do this kind of thing in the router. Inside the router, you don't have these data when querying.
So, move this logic into Room Component or something similar.